packages/styles/settings-tools/_all-settings.scss

```diff
@@ -20,3 +20,5 @@
 @import './_s.links.scss';
 @import './_s.select.scss';
 @import './_s.file-uploader.scss';
+@import './_s.stars.scss';
+@import './_u.screen-reader.scss';
```

Comment:
A mozaic component should not have utilities as dependencies, because we can't make sure the user will import it. Simple components should be dependency free. In my tag PR I created a mixing for that. Open to discussion.

Reply:
Fixed